_config.yml

```diff
@@ -35,7 +35,7 @@ author:
   photo                     : "uploads/SSE_LOGO.png"
   photo2x                   : "uploads/SSE_LOGO.png"
 
-
+APIBaseUrl: "http://127.0.0.1:5000"
```

Comment:
Is this something you'd anticipate us replacing later on in the future through an environment variable? Just curious.